doc/quickstart.md

```diff
@@ -199,3 +199,9 @@ To build the binary.
 ```bash
 make build
 ```
+
+If you want to test changes to the `network config daemon`, you must:
+- build and tag an image locally with `docker build -f Dockerfile.sriov-network-config-daemon -t imagename`
+- change `hack/env.sh` value for `SRIOV_NETWORK_CONFIG_DAEMON_IMAGE` pointing _imagename_
```

Comment:
I think you also need to push the image to a registry, where the worker node can pull it.

Reply:
That's correct (and that's what I did), will update.